src/test/java/org/example/automation/amazon/AddToRemoveFromCartTest.java

```diff
@@ -0,0 +1,93 @@
+package org.example.automation.amazon;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class AddToRemoveFromCartTest extends BaseWebDriverTest {
+    public static final String SITE_URL = "https://www.amazon.com";
+    public static final String ADDED_TO_CART = "Added to Cart";
+    public static final String ZERO_VALUE = "$0.00";
+    public static final String YOUR_AMAZON_CART_IS_EMPTY = "Your Amazon Cart is empty.";
+    public static final String CART_COUNT = "1";
+    public static final int WAIT_SECONDS = 10;
+
+    @FindBy(xpath = "//div[contains(@class,\"a-section a-spacing-none fluid-image-container\")]//img")
```

Comment:
It is better to put all elements into special classes of your PageObject. But it is your next task. Hope to see these fields in page classes